force-app/main/default/classes/nksGetStoUtilityController.cls

```diff
@@ -230,4 +249,25 @@ public without sharing class nksGetStoUtilityController {
             LIMIT 1
         ];
     }
+
+    public class GroupedSkills {
+        @AuraEnabled
+        public List<ServiceResourceSkill> skillList;
+        public List<Id> skillIdList;
+        @AuraEnabled
+        public List<ServiceResourceSkill> userConfidentialitySkills;
+        public List<Id> userConfidentialitySkillIds;
+        @AuraEnabled
+        public List<ServiceResourceSkill> userOriginSkills;
+        public List<String> userOriginSkillIds;
+
+        public GroupedSkills() {
+            skillList = new List<ServiceResourceSkill>();
+            skillIdList = new List<Id>();
+            userConfidentialitySkills = new List<ServiceResourceSkill>();
+            userConfidentialitySkillIds = new List<Id>();
+            userOriginSkills = new List<ServiceResourceSkill>();
+            userOriginSkillIds = new List<String>();
+        }
```

Comment:
Jeg syns dette ble litt rotete med én liste for skill og én for id for hver skilltype. Kan du ikke heller erstatte disse 6 listene med 3 maps med SkillId og DeveloperName?